packages/server/prisma/schema.prisma

```diff
@@ -12,16 +12,42 @@ datasource db {
 }
 
 model CategoryDataBase {
-  id                   Int                   @id @default(autoincrement())
+  id                   String                @id @default(uuid())
   rataextraRequestPage String
-  alfrescoFolder       String
+  alfrescoFolder       String                @unique
   categoryDataContents CategoryDataContents?
+  categoryComponents   CategoryComponent[]
   writeRights          String
 }
 
 model CategoryDataContents {
-  id               Int              @id @default(autoincrement())
+  id               String           @id @default(uuid())
   categoryDataBase CategoryDataBase @relation(fields: [baseId], references: [id])
-  baseId           Int              @unique
+  baseId           String           @unique
   fields           Json
 }
+
+model CategoryComponent {
+  id               String           @id @default(uuid())
+  categoryDataBase CategoryDataBase @relation(fields: [categoryId], references: [id])
+  categoryId       String
+  node             Node?
+  card             Card?
+}
+
+model Node {
+  id                  String             @id @default(uuid())
+  title               String
+  type                String
+  nodeId              String
```

Comment:
What's the difference between Node's id and nodeId?

Reply:
Idea is that `nodeId` represents Alfresco nodeId and `id` is just id for this table.

Reply:
alfrescoNodeId should be best 👍

Reply:
Let's go with `alfrescoNodeId`!

No, CategoryDataContents `baseId` is used [as foreign key to create relation](https://www.prisma.io/docs/concepts/components/prisma-schema/relations) with `CategoryDataBase`. alfrescoNodeId is just value of alfresco  node.